x-pack/plugins/apm/public/components/shared/KueryBar/Typeahead/Suggestions.js

```diff
@@ -53,12 +52,12 @@ class Suggestions extends Component {
   }
 
   render() {
-    if (!this.props.show || isEmpty(this.props.suggestions)) {
+    if (!this.props.show || this.props.suggestions.length === 0) {
```

Comment:
I don't mind this change but curious to hear why you did it. Should we stop using `isEmpty` because it's lodash 3?

Reply:
this change was based on a suggestion from me. I didn't realise this was an apm file.
I noticed that `isEmpty` was being included only for this check, and as `suggestions` is an array, i felt it was unnecessary to use a 3rd party function to check its length.

Reply:
actually, this change may have been made in error.
my comment was for an ML file.
https://github.com/elastic/kibana/pull/31763/files#r260821761

Reply:
Yes, thanks for clarifying @jgowdyelastic 👍 
I've gone ahead and removed the change @sqren - sorry about that! 😄